### ISSUE TITLE
Error in to_ascii()

### ISSUE THREAD

**Issue by stuarteberg** [omegacen/conda-depgraph#4]
It seems that the latest version of `conda-depgraph` no longer works, possibly due to changes in the `asciinet` dependency.

I installed it from scratch, and then tried an example from the README:

```
$ conda install -y -n base -c omegacen conda-depgraph

$ conda depgraph --name=base out conda
Traceback (most recent call last):
  File "/opt/miniconda/bin/conda-depgraph", line 10, in <module>
    sys.exit(main())
  File "/opt/miniconda/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/opt/miniconda/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/opt/miniconda/lib/python3.7/site-packages/click/core.py", line 1164, in invoke
    return _process_result(rv)
  File "/opt/miniconda/lib/python3.7/site-packages/click/core.py", line 1102, in _process_result
    **ctx.params)
  File "/opt/miniconda/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/opt/miniconda/lib/python3.7/site-packages/conda_depgraph/cli.py", line 54, in process_subcommands
    o = of(g)
  File "/opt/miniconda/lib/python3.7/site-packages/conda_depgraph/outputs.py", line 9, in to_ascii
    a = graph_to_ascii(g)
  File "/opt/miniconda/lib/python3.7/site-packages/asciinet/__init__.py", line 108, in graph_to_ascii
    return _asciigraph.graph_to_ascii(graph, timeout=timeout).decode(encoding='UTF-8')
AttributeError: 'str' object has no attribute 'decode'
```

PS -- I love this tool.  I had it installed on my old machine, where it worked well.  Now I'm trying to re-install it on my new machine, in a new miniconda installation.

**Comment by teake** [comment]
Thanks for the kind words!

Unfortunately I can't reproduce this on CentOS 7.7 with conda 4.8.2. Could you post the output of both `conda info` and `conda list -n base`?

**Comment by stuarteberg** [comment]
Sure!  Here's the summary. Click below for the full info.

```
          conda version : 4.8.2
         python version : 3.7.6.final.0
               platform : osx-64
```

```
$ conda list -n base | grep -E 'python |asciinet |click |conda |networkx |depgraph '
asciinet                  0.4.pre.cec4d1f               0    omegacen
click                     7.0                        py_0    conda-forge
conda                     4.8.2                    py37_0    conda-forge
conda-depgraph            1.1.1                         0    omegacen
msgpack-python            1.0.0            py37ha1b3eb9_0    conda-forge
networkx                  2.4                        py_0    conda-forge
python                    3.7.6                h5c2c468_2    conda-forge
```

<details>

<summary>Full conda info</summary>

```
$ conda info

     active environment : base
    active env location : /opt/miniconda
            shell level : 1
       user config file : /Users/bergs/.condarc
 populated config files : /opt/miniconda/.condarc
          conda version : 4.8.2
    conda-build version : 3.18.11
         python version : 3.7.6.final.0
       virtual packages : __osx=10.15.3
       base environment : /opt/miniconda  (writable)
           channel URLs : https://conda.anaconda.org/flyem-forge/osx-64
                          https://conda.anaconda.org/flyem-forge/noarch
                          https://conda.anaconda.org/conda-forge/osx-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /opt/miniconda/pkgs
                          /Users/bergs/.conda/pkgs
       envs directories : /opt/miniconda/envs
                          /Users/bergs/.conda/envs
               platform : osx-64
             user-agent : conda/4.8.2 requests/2.22.0 CPython/3.7.6 Darwin/19.3.0 OSX/10.15.3
                UID:GID : 503:20
             netrc file : None
           offline mode : False
```

</details>

<details>

<summary>Full base environment</summary>

```
$ conda list -n base
# packages in environment at /opt/miniconda:
#
# Name                    Version                   Build  Channel
anaconda-client           1.7.2                      py_0    conda-forge
argcomplete               1.11.1                     py_0    conda-forge
asciinet                  0.4.pre.cec4d1f               0    omegacen
attrs                     19.3.0                     py_0    conda-forge
beautifulsoup4            4.8.2                    py37_0    conda-forge
bzip2                     1.0.8                h0b31af3_2    conda-forge
ca-certificates           2019.11.28           hecc5488_0    conda-forge
certifi                   2019.11.28               py37_0    conda-forge
cffi                      1.13.2           py37h33e799b_0    conda-forge
chardet                   3.0.4                 py37_1003    conda-forge
click                     7.0                        py_0    conda-forge
clyent                    1.2.2                      py_1    conda-forge
conda                     4.8.2                    py37_0    conda-forge
conda-build               3.18.11                  py37_0    conda-forge
conda-depgraph            1.1.1                         0    omegacen
conda-package-handling    1.6.0            py37h0b31af3_1    conda-forge
conda-verify              3.1.1                 py37_1000    conda-forge
cryptography              2.8              py37hafa8578_1    conda-forge
curl                      7.65.3               h22ea746_0    conda-forge
decorator                 4.4.1                      py_0    conda-forge
expat                     2.2.5             h4a8c4bd_1004    conda-forge
filelock                  3.0.10                     py_0    conda-forge
future                    0.18.2                   py37_0    conda-forge
gettext                   0.19.8.1          h46ab8bc_1002    conda-forge
git                       2.24.0          pl526hdc91d69_1    conda-forge
glob2                     0.7                        py_0    conda-forge
icu                       64.2                 h6de7cb9_1    conda-forge
idna                      2.8                   py37_1000    conda-forge
importlib_metadata        1.4.0                    py37_0    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
jinja2                    2.10.3                     py_0    conda-forge
jsonschema                3.2.0                    py37_0    conda-forge
jupyter_core              4.6.1                    py37_0    conda-forge
krb5                      1.16.4               h1752a42_0    conda-forge
libarchive                3.3.3             h8912c15_1007    conda-forge
libcurl                   7.65.3               h16faf7d_0    conda-forge
libcxx                    9.0.1                         1    conda-forge
libedit                   3.1.20170329      hcfe32e1_1001    conda-forge
libffi                    3.2.1             h6de7cb9_1006    conda-forge
libiconv                  1.15              h01d97ff_1005    conda-forge
liblief                   0.9.0                h3e78482_1    conda-forge
libssh2                   1.8.2                hcdc9a53_2    conda-forge
libxml2                   2.9.10               h53d96d6_0    conda-forge
lz4-c                     1.8.3             h6de7cb9_1001    conda-forge
lzo                       2.10              h1de35cc_1000    conda-forge
markupsafe                1.1.1            py37h0b31af3_0    conda-forge
more-itertools            8.1.0                      py_0    conda-forge
msgpack-python            1.0.0            py37ha1b3eb9_0    conda-forge
natsort                   7.0.1                      py_0    conda-forge
nbformat                  5.0.3                      py_0    conda-forge
ncurses                   6.1               h0a44026_1002    conda-forge
networkx                  2.4                        py_0    conda-forge
openjdk                   11.0.1            hbbe82c9_1018    conda-forge
openssl                   1.1.1d               h0b31af3_0    conda-forge
pcre                      8.43                 h4a8c4bd_0    conda-forge
perl                      5.26.2            haec8ef5_1006    conda-forge
pip                       19.3.1                   py37_0    conda-forge
pkginfo                   1.5.0.1                    py_0    conda-forge
psutil                    5.6.7            py37h0b31af3_0    conda-forge
publish-conda-stack       0.2.4                      py_0    flyem-forge
py-lief                   0.9.0            py37h0ceac7d_1    conda-forge
pycosat                   0.6.3           py37h0b31af3_1002    conda-forge
pycparser                 2.19                     py37_1    conda-forge
pyopenssl                 19.1.0                   py37_0    conda-forge
pyrsistent                0.15.7           py37h0b31af3_0    conda-forge
pysocks                   1.7.1                    py37_0    conda-forge
python                    3.7.6                h5c2c468_2    conda-forge
python-dateutil           2.8.1                      py_0    conda-forge
python-libarchive-c       2.9                      py37_0    conda-forge
python.app                1.2             py37h0b31af3_1201    conda-forge
pytz                      2019.3                     py_0    conda-forge
pyyaml                    5.3              py37h0b31af3_0    conda-forge
readline                  8.0                  hcfe32e1_0    conda-forge
requests                  2.22.0                   py37_1    conda-forge
ripgrep                   11.0.2               h01d97ff_3    conda-forge
ruamel_yaml               0.15.80         py37h0b31af3_1000    conda-forge
setuptools                45.0.0                   py37_1    conda-forge
six                       1.14.0                   py37_0    conda-forge
soupsieve                 1.9.4                    py37_0    conda-forge
sqlite                    3.30.1               h93121df_0    conda-forge
tk                        8.6.10               hbbe82c9_0    conda-forge
tqdm                      4.41.1                     py_0    conda-forge
traitlets                 4.3.3                    py37_0    conda-forge
urllib3                   1.25.7                   py37_0    conda-forge
wheel                     0.33.6                   py37_0    conda-forge
xz                        5.2.4             h1de35cc_1001    conda-forge
yaml                      0.2.2                h0b31af3_1    conda-forge
zipp                      2.0.0                      py_0    conda-forge
zlib                      1.2.11            h0b31af3_1006    conda-forge
zstd                      1.4.4                he7fca8b_1    conda-forge
```

</details>

**Comment by teake** [comment]
Seems like the problem was due to a newer msgpack-python version. I haven't pinned this properly in the asciinet recipe, and as a result you're getting the freshly released 1.0.0 whereas asciinet only works with older versions.

To fix this, simply do
```
$ conda install -c conda-forge --name base "msgpack-python=0.6"
```
and you should be good. I'll update the asciinet recipe so this doesn't happen again.

**Comment by stuarteberg** [comment]
Yep, that fixes it for me.  Thanks!
